docs/examples/201/servicebus-create-queue/main.bicep

```diff
@@ -0,0 +1,32 @@
+param serviceBusNamespaceName string
+param serviceBusQueueName string
+param location string = resourceGroup().location
+
+resource serviceBusNamespace 'Microsoft.ServiceBus/namespaces@2017-04-01' = {
+  name: serviceBusNamespaceName
+  location: location
+  sku: {
+    name: 'Standard'
+  }
+  properties: {}
+}
+
+resource serviceBusQueue 'Microsoft.ServiceBus/namespaces/queues@2017-04-01' = {
+  name: format('{0}/{1}', serviceBusNamespaceName, serviceBusQueueName)
```

Comment:
use string interpolation and get the name from the resource identifier to establish a dependency:

```
name: '${serviceBusNamespace.name}/${serviceBusQueueName}'
```